charts/budibase/templates/minio-service-deployment.yaml

```diff
@@ -42,6 +42,11 @@ spec:
             secretKeyRef:
               name: {{ template "budibase.fullname" . }}
               key: objectStoreSecret
+        {{ if .Values.services.tlsRejectUnauthorized }}
```

Comment:
Don't think we need this in the proxy, redis or minio deployments since it's Node specific